rubin_scheduler/scheduler/features/conditions.py

```diff
@@ -85,8 +85,8 @@ def __init__(
         bulk_cloud : `float`
             The fraction of sky covered by clouds. (In the future might
             update to transparency map)
```

Comment:
This description might need updating?
I'm guessing that cloud_maps vs. bulk_cloud is either/or right now, but we still want to support both. It might be good to add some more explicit documentation of this.